plugins/modules/ini_file.py

```diff
@@ -44,6 +44,30 @@
       - If being omitted, the O(option) will be placed before the first O(section).
       - Omitting O(section) is also required if the config format does not support sections.
     type: str
+  section_has_values:
+    type: list
+    elements: dict
+    required: false
+    suboptions:
+      option:
+        type: str
+        description: Matching O(section) must contain this option.
+        required: true
+      value:
+        type: str
+        description: Matching O(option) must have this specific value.
```

Comment:
This is referring to `option` in `section_has_values[]`, not to the top-level `option`:
```suggestion
        description: Matching O(section_has_values[].option) must have this specific value.
```